lib/screens/explore_screens/vote_screens/vote/interactor/viewmodels/vote_state.dart

```diff
@@ -8,8 +8,7 @@ class VoteState extends Equatable {
   final bool shouldShowDelegateIcon;
   final bool isCitizen;
   final List<CategoryDelegate> currentDelegates;
-
-  bool get voteCycleHasEnded => cycleEndTimestamp < DateTime.now().millisecondsSinceEpoch;
+  final bool voteCycleHasEnded;
```

Comment:
needed to make this part of the state after all... as it is part of the state...